src/sbt-test/sbt-dependency-graph/intervalRangedVersions/build.sbt

```diff
@@ -1,5 +1,7 @@
 scalaVersion := "2.9.1"
 
+resolvers += "typesafe maven" at "http://repo.typesafe.com/typesafe/maven-releases/"
```

Comment:
You should always use https for Maven repos: http://www.ontoillogical.com/blog/2014/07/28/how-to-take-over-any-java-developer/